yang/ietf-schedule.yang

```diff
@@ -91,20 +91,25 @@ module ietf-schedule {
             + ':[0-5][0-9]:[0-5][0-9]))|P([0-9]+)W';
     }
     description
-      "Duration of the time. The format can represent nominal
-       durations (weeks and days) and accurate durations (hours, 
-       minutes, and seconds). Note that this value type doesn't 
-       support the 'Y' and 'M' designators to specify durations in 
-       terms of years and months. Negative durations are typically 
-       used to schedule an alarm to trigger before an associated time.";
+      "Duration of the time as defined in Section 3.3.6 of RFC 5545. 
+       The format can represent nominal durations (weeks designated 
+       by 'W' and days designated by 'D') and accurate durations 
+       (hours:minutes:seconds follows the designator 'T').
+           
+       Note that this value type doesn't support the 'Y' and 'M' 
+       designators to specify durations in terms of years and months. 
+       
+       Negative durations are typically used to schedule an alarm to
```

Comment:
Added a ref to [Section 3.8.6.3](https://www.rfc-editor.org/rfc/rfc5545#section-3.8.6.3) because otherwise we will need to elaborate more.

Reply:
Sure, understood.